virtManager/config.py

```diff
@@ -212,6 +212,17 @@ def embeddable_graphics(self):
         ret = ["vnc", "spice"]
         return ret
 
+    def get_perfered_graphics_index(self, gdevs):
```

Comment:
spelling is `preferred`

Reply:
This function was removed. Thanks very much!